src/docbuilder/rustwide_builder.rs

```diff
@@ -468,6 +473,12 @@ impl RustwideBuilder {
             cargo_args.push(target);
         };
 
+        let tmp_jobs;
+        if let Some(cpu_limit) = self.cpu_limit {
+            tmp_jobs = format!("-j{}", cpu_limit);
+            cargo_args.push(&tmp_jobs);
```

Comment:
If we're adding a hard limit in rustwide, why are we also telling cargo to slow down? Is the concern that cargo will detect the true number of CPUs and try to spawn too many threads?

Reply:
You don't want unnecessary contention over nonexistent resources yes.